hypothesis_2.ipynb

```diff
@@ -223,6 +234,7 @@
     "    # update row columns after cleaning\n",
     "    row['langs'] = langs\n",
     "    row['levels'] = levels\n",
+    "    # AW: should we unset(row['languages']) ?\n",
```

Comment:
@adamwight 
[Solved] I was not deleting it since it is highly useful for references of what the new data should look like. Now it feels safe.

Reply:
I see!  Sure, no harm in keeping it, I wasn't sure if that was intentional or not.